model-api/bpmn-model/src/test/java/org/camunda/bpm/model/bpmn/builder/ProcessBuilderTest.java

```diff
@@ -103,9 +103,22 @@ public void testCreateEmptyProcess() {
       .hasSize(1);
 
     Process process = (Process) processes.iterator().next();
+
     assertThat(process.getId()).isNotNull();
   }
 
+  @Test
+  public void emptyProcessShouldHaveDefaultHTTL() {
+    modelInstance = Bpmn.createProcess().done();
+
+    var process = (Process) modelInstance.getModelElementsByType(processType)
+        .iterator()
+        .next();
+
+    assertThat(process.getCamundaHistoryTimeToLiveString())
+        .isEqualTo("P180D");
```

Comment:
🔧 You could also use the actual constant.

```suggestion
        .isEqualTo(ProcessImpl.DEFAULT_HISTORY_TIME_TO_LIVE);
```